src/DbLocalizationProvider.MigrationTool/XmlDocumentParser.cs

```diff
@@ -57,28 +63,33 @@ public ICollection<LocalizationResource> ReadXml(XDocument xmlDocument)
                         continue;
                     }
 
-                    var existingResource = result.FirstOrDefault(r => r.ResourceKey == resourceKey);
+                    var existingResource = result.FirstOrDefault(r => r.ResourceKey == resourceKey.ToLower());
 
                     if (existingResource != null)
                     {
                         var existingTranslation = existingResource.Translations.FirstOrDefault(t => t.Language == cultureName);
 
                         if (existingTranslation != null)
                         {
-                            throw new NotSupportedException($"Found duplicate translations for resource with key: {resourceKey} for culture: {cultureName}");
+                            if (!ignoreDuplicateKeys)
+                            { 
+                                throw new NotSupportedException($"Found duplicate translations for resource with key: {resourceKey} for culture: {cultureName}");
+                            }
+                        }
+                        else
+                        { 
+                            existingResource.Translations.Add(new LocalizationResourceTranslation
+                                                              {
+                                                                  Language = cultureName,
+                                                                  Value = resourceTranslation
+                                                              });
                         }
-
-                        existingResource.Translations.Add(new LocalizationResourceTranslation
-                                                          {
-                                                              Language = cultureName,
-                                                              Value = resourceTranslation
-                                                          });
                     }
                     else
                     {
                         var resourceEntry = new LocalizationResource
                                             {
-                                                ResourceKey = resourceKey,
+                                                ResourceKey = resourceKey.ToLower(),
```

Comment:
I'm sure about this line.. It might cause problems later to look up this resource as not yet everywhere resource key is made case insensitive

Reply:
I was under the assumption that once in epi keys are case insensitive because trying to import a json with same key with different case throws an exception. If that is true it should not matter that all keys are made lowercase.
Is it intended that keys are case sensitive? In that case the current behaiviour is correct and the import should be modified to allow it.

Reply:
case sensitivity is for generated keys from code (for now). this will be changed in next major version.
is it possible for you to try out to import key in UPPER case and call Episerver API with lower? This might collide with use case when migration is done for "normal" resources (like with key `Namespace.Class.Property`)